src/main/java/uk/gov/companieshouse/pscfiling/api/model/FilingDescription.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.pscfiling.api.model;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class FilingDescription {
+    @Value("${description.psc07}")
+    private String psc07;
+
+    // Spring no args constructor
+    public FilingDescription() {
```

Comment:
Constructor definition is not needed; should be removed.

Reply:
removed... at one point as I was writing this, it was needed but no longer.